CHANGES.md

```diff
@@ -39,6 +39,9 @@ Specification language, type inference
   solved by writing an explicit `return` clause, sometimes even simply
   an explicit `return _` clause.
 
+- The commands `Existential` and `Grab Existential` have been
```

Comment:
The command is `Grab Existential Variables`, and `xxx` should be fixed before merging